packages/aws-cdk-lib/aws-apigateway/lib/restapi.ts

```diff
@@ -298,6 +299,28 @@ export interface SpecRestApiProps extends RestApiBaseProps {
    * @default - Compression is disabled.
    */
   readonly minCompressionSize?: Size;
+
+  /**
+   * This property applies only when you use OpenAPI to define your REST API.
+   * The Mode determines how API Gateway handles resource updates.
```

Comment:
Moving the property to SpecRestApiProps would make this explanation simpler.

```suggestion
   * The Mode that determines how API Gateway handles resource updates.
```